command.go

```diff
@@ -267,6 +267,7 @@ func parseEnvArgs(args []string) (Command, error) {
 	flag.String("assume", "", "Role to assume")
 	flag.Bool("no-session", false, "Disable use of temporary credentials")
 	flag.Bool("refresh", false, "Start a new session with new temporary credentials and a refreshed expiration")
+	flag.String("region", os.Getenv("AWS_DEFAULT_REGION"), "The AWS region to use to generate STS credentials")
```

Comment:
Isn't `AWS_REGION` the preferred variable here with this being the fallback?

Reply:
After talking with some AWS folks, it appears that there isn't a consensus on which variable should be used (some tools completely ignore one variable or another), but I think I agree with your premise and will update the code to prefer `AWS_REGION` and fall back to `AWS_DEFAULT_REGION`.